### PR TITLE
Created installation.txt

### PR DESCRIPTION
**Pull request by LakshitF**
Moved the development section to installation.txt as requested.

### CODE REVIEW

**Comment by s-spindler** [comment]
@LakshitF Thanks for the contribution! Can you please change the file name to DEVELOPMENT.md? Also, the section should then be removed from the README.md.

File: installation.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Development
+
```
Comment (s-spindler) [inline installation.txt]: Maybe super nitpicky, but the second empty line is redundant.

File: installation.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Development
```
Comment (s-spindler) [inline installation.txt]: Since this is now a separate file, it should be a level 1 header, so only one `#`.

**Comment by s-spindler** [comment]
@LakshitF Sorry for late feedback; can you please also implement the other feedback suggestions?

**Comment by LakshitF** [comment]
Have done.

**Review by s-spindler — CHANGES_REQUESTED** [review]

**Comment by s-spindler** [comment]
@LakshitF Sorry, I hadn't submitted the review :( Please see above...

**Pull request closed without merging**